rct229/rulesets/ashrae9012019/data/ashrae_90_1_table_G3_1_1_2.json

```diff
@@ -0,0 +1,148 @@
+{
+    "Baseline_Service_Water_Heating_System": [
+        {
+            "building_area_type": "Automotive facility",
```

Comment:
I don't think the str would be the description of the `ServiceWaterHeatingSpaceOptions2019ASHRAE901`, but the enums.
So for example, the `Automotive facility` here, should be `AUTOMOTIVE_FACILITY`

Reply:
Addressed.